lib/ansible/modules/system/pamd.py

```diff
@@ -405,15 +402,19 @@ def add_module_arguments(service, old_rule, module_args):
     result = {'action': 'args_present'}
     changed = False
     change_count = 0
-    if isinstance(module_args, ansible.module_utils.six.string_types):
-        module_args = module_args.split(' ')
 
     for rule in service.rules:
         if (old_rule.rule_type == rule.rule_type and
                 old_rule.rule_control == rule.rule_control and
                 old_rule.rule_module_path == rule.rule_module_path):
             for arg_to_add in module_args:
-                if "=" in arg_to_add:
+                if arg_to_add not in rule.rule_module_args:
```

Comment:
If the pamd line was
```
password    sufficient    pam_unix.so  remember=3
```
and you try to change it to remember=5, won't doing this condition first result in:
```
password    sufficient    pam_unix.so  remember=3 remember=5
``` ?

Is this what we want?  or do we want this instead:
```
password    sufficient    pam_unix.so  remember=5
``` ?